channel/types/channel_state.go

```diff
@@ -0,0 +1,29 @@
+package types
+
+import (
+	"perun.network/go-perun/channel"
+	"reflect"
+)
+
+type ChannelState struct {
```

Comment:
I would like a short comment that this is the `ChannelState` coming from the Perun SDK.